src/DIRAC/ConfigurationSystem/Service/TornadoConfigurationHandler.py

```diff
@@ -48,7 +48,7 @@ def export_getCompressedData(self):
       Returns the configuration
     """
     sData = self.ServiceInterface.getCompressedConfigurationData()
-    return S_OK(b64encode(sData))
+    return S_OK(b64encode(sData).decode('utf-8'))
```

Comment:
@chrisburr can confirm, but I was under the impression that we should just use `decode()`

Reply:
Yup, it's the default in Python 3: https://docs.python.org/3/library/stdtypes.html#bytes.decode

(let's not think about Python 2 too much but `b64encode` will only ever return ASCII data so it doesn't matter)